src/css/application.css

```diff
@@ -517,6 +517,22 @@ body {
   opacity: 0.5;
 }
 
+.preview__reset-button {
+  background: url(../images/popout.png);
```

Comment:
Looks like FontAwesome [does have a refresh icon](http://fontawesome.io/icon/refresh/) and we could also switch the existing [pop-out icon to FontAwesome](http://fontawesome.io/icon/external-link/) while we’re in there